next-docs/pages/templates/typescript/contracts.mdx

```diff
@@ -1 +1,453 @@
-# Contracts
+# Contracts (Vanilla and React)
+
+This section explains the contracts for the vanilla and react templates.
+The Three.JS template uses diffeerent contracts.
+
+The onchain components can be divided into two types of functionality:
+
+- **Data**, the part of the system that stores the information.
+- **Logic**, the part of the system that can be called to read or modify the data.
+
+## Data
+
+### The data schema
+
+#### `mud.config.ts`
+
+The data schema is declared in `packages/contracts/mud.config.ts`.
```

Comment:
I feel like "table schema" would be more precise here (especially if we go with the suggestion above)

Reply:
Done.